content/cpi-api-v2-method/create-vm.md

```diff
@@ -60,14 +60,52 @@ Most CPIs choose to communicate with the default Agent. Hence, the communication
     "ephemeral": "/dev/sdb",
     "persistent": {}
   },
-
-  "mbus": "https://mbus:mbus-password@0.0.0.0:6868"
-  "ntp": [ "0.pool.ntp.org", "1.pool.ntp.org" ],
+  "ntp": null,
+  "mbus": "",
   "blobstore": {
-    "provider": "local",
-    "options": { "blobstore_path": "/var/vcap/micro_bosh/data/cache" }
+    "provider": "",
+    "options": {}
   },
-  "env": {}
+  "env": {
+    "bosh": {
+      "password": "",
```

Comment:
I would be a little hesitant on trying to document all of these properties. It might be useful to keep an example like `mbus` just to help reinforce structure of the hash, but I would be a little concerned on 1) trying to keep values in sync with what's documented in [`env`](https://bosh.io/docs/manifest-v2/#instance-groups) and 2) CPI authors seeing these enumerated and relying on the contents for some reason.

Reply:
Regarding 1): I agree that it is not optimal trying to keep those in sync and that the manifest-v2 docs would be the more intuitive place for release authors. But how would you suggest to document the `env.bosh.` properties which are only relevant for `bosh create-env` without a registry then? Since they were removed from the CPI's `cloud_provider` section, there is no documentation for CPI authors anymore.

Reply:
I think @voelzmo has already given some thoughts to that